### ISSUE TITLE
Bower package name as lowercase

### ISSUE THREAD

**Issue by stigok** [Dogfalo/materialize#3012]
A problem arised when I had already installed `Materialize ~0.97.6` in my repo, then angular-materialize has dependency on `materialize ~0.97.2` (all lowercase). They end up being installed side by side as different packages, resulting in this dir tree:

```
bower_components
├── materialize
└── Materialize
```

... which cordova did not like (which is how I spotted this). However, attempting to install with `bower install Materialize` yields _Package Materialize not found_.

Would it be good to name the package with all lowercase letters in bower.json? This may be a deeper issue with bower as well.

### Reproduce
```
bower install angular-materialize
bower install materialize

bower_components
├── angular
├── angular-materialize
├── jquery
├── materialize
└── Materialize

```

**Comment by jacobq** [comment]
Would this issue be resolved if materialize was always lowercase? If so, then this may be a duplicate of
https://github.com/Dogfalo/materialize/issues/2389
and should be fixed by this PR
https://github.com/Dogfalo/materialize/pull/3087

**Comment by kevgathuku** [comment]
Another issue referencing this is #3195 which shows the warning displayed by bower
I've also included a PR to fix both the naming and the other warnings in #3212

**Comment by Dogfalo** [comment]
Fixed by #3212
